isala.local/app/controllers/changepassword.php

```diff
@@ -32,6 +33,7 @@ public function index() // TODO: don't transfer password over URL
                 if (strlen($this->err_msg) < 1) echo "<p style=\"color: #FC240F\">Something went wrong</p>";
                 else echo "<p style=\"color: #FC240F\">" .  htmlentities($this->err_msg) . "</p>";
             } else {
+                logger::log($_SESSION['uid'], 'Password change failed', $this->model);
```

Comment:
$this->err_msg is beschikbaar, zou meer informatie bieden als deze ook in de message wordt verwerkt.
Ik zie ook geen log voor wanneer de validatie van het wachtwoord correct is maar het alsnog verkeerd gaat (attemptPasswordChange  == false).